### ISSUE TITLE
Group 'right_arm' not found in model 'baxter'

### ISSUE THREAD

**Issue by ysl208** [ysl208/iRoPro#57]
This could be because you launched the simulation and then tried the real robot, which both have different URDFs. So need to overwrite the old URDF. launch :
``` 
roslaunch rapid_pbd baxter_moveit.launch  right_electric_gripper:=true left_electric_gripper:=true
```
URDF is in `../src/baxter_common/baxter_description/urdf/baxter.urdf.xacro`
URDF copied into `/local/collada` folder
SRDF is in `src/moveit_robots/baxter/baxter_moveit_config/config`

check https://groups.google.com/forum/#!topic/moveit-users/gPJtzIX68yw

**Comment by ysl208** [comment]
Same problem occurred when setting up UR5 and using oja model

```
Group 'arm' not found in model 'oja'
```
Edit: this was caused by not using the correct move_group.launch and planning_context.launch as the robot moveit srdf. Always make sure that you use the same ones!

**Comment by ysl208** [comment]
fixed in c94e4f1e5121f9189107b0a5d3853b2993045d22
